### PR TITLE
Add warning in dev/int/ci/infra environments

### PR DESCRIPTION
**Pull request by gjn**
This adds a clearly visible warning below the search box when the application does not run under the official host. This is the sensibilize people not to share a technical link.

[Testklink](https://mf-geoadmin3.int.bgdi.ch/gjn_devint/index.html)

### CODE REVIEW

File: src/index.mako.html [hunk 1]
```diff
@@ -154,6 +154,9 @@
   % if device == 'desktop':
         <span ga-help="24,31,25" ga-help-options="{showOnHover:true}"></span>
   % endif
+        <div class="warning-not-prod" ng-class="{'host-not-prod': !globals.hostIsProd}">
```
Comment (oterral) [inline src/index.mako.html]: Can we use ng-if instead : `ng-if=::!globals.hostIsProd`  ?

File: src/style/app.less [hunk 2]
```diff
@@ -1930,7 +1930,35 @@ body:not(.embed) {
...
 
+  &:hover, &:focus, &:active {
+    z-index: 3000;
+    height: auto;
+    white-space: normal;
+  }
+
+  &.host-not-prod {
+    display: block;
+  };
```
Comment (oterral) [inline src/style/app.less]: I think you can optimize the css.

**Comment by oterral** [comment]
I add minor comments otherwise I think it's a good idea.

**Comment by procrastinatio** [comment]
Yes, but too small. Should be in BIG LETTERS accross the map.

And also add something like "Draft" on the print, in case people are printing not yet published data.

File: src/index.mako.html (hunk 1 above)
Comment (gjn) [inline src/index.mako.html]: yes, will adapt

File: src/style/app.less [hunk 3]
```diff
@@ -1930,7 +1942,35 @@ body:not(.embed) {
...
+    z-index: 3000;
+    height: auto;
+    white-space: normal;
+  }
 
+  &.host-not-prod {
+    display: block;
+  };
+
+}
```
Comment (gjn) [inline src/style/app.less]: css is taken from 3d warning that exists already.

**Comment by gjn** [comment]
I will add a big text in the logo. I thought about text across the map, but I think it's to invasive. We work with this daily and it'll be used by externals for abnahme. Have to find balance between clear signal and still having workable stuff.

For print: in general yes, can be added later on.

**Comment by oterral** [comment]
Just to mention writing something on the official logo is not CD bund compliant I think :).

**Comment by gjn** [comment]
You are right. It's also not an officiel site :)

**Pull request merged**